AirCasting/SDSync/Parsing/SDCardMeasurementsParser.swift

```diff
@@ -48,6 +48,26 @@ class SDCardMeasurementsParser {
                                      pm2_5: pm2_5,
                                      pm10: pm10)
     }
+    
+    func getUUID(lineString: String) -> String? {
+        let measurementInfo = lineString.split(separator: ",")
+        guard measurementInfo.count == 13 else {
+            Log.warning("Line corrupted: \(lineString)")
+            return nil
+        }
+        
+        return String(measurementInfo[SDCardCSVFileFactory.Header.uuid.rawValue])
+    }
+    
+    func getMeasurementTime(lineString: String) -> Date? {
+        let measurementInfo = lineString.split(separator: ",")
+        guard measurementInfo.count == 13 else {
```

Comment:
`13` here is:
1) Duplicated in both functions
2) A magic number

For those reasons I suggest moving it to a `let` somewhere and name it well